i18n/zh/docusaurus-plugin-content-docs/current.json

```diff
@@ -123,6 +123,10 @@
     "message": "GreptimeDB 企业版",
     "description": "The label for category GreptimeDB Enterprise in sidebar docs"
   },
+  "sidebar.docs.doc.enterprise/console-ui": {
+    "message": "企业版管理控制台",
+    "description": "The label for Enterprise Management Console doc in sidebar docs"
+  },
```

Comment:
```suggestion
```

The document item do not need to translate. It will use the h1 title inside the article as the navigation title.